src/components/elements/side-tab.js

```diff
@@ -5,23 +5,35 @@ import { navigate } from '@reach/router'
 import Wrapper from '../containers/wrapper'
 import { Text } from './typography'
 import { getLocationHash } from 'common/utility'
+import device, { size } from 'themes/device'
+import { Desktop, Mobile } from 'components/containers/show'
+import Path from 'images/svg/path.svg'
 
 const StyledSideTab = styled(Wrapper)`
     padding: 0;
     display: flex;
+
+    @media ${device.tabletL} {
+        justify-content: center;
+        flex-direction: column;
+        width: 100%;
+    }
 `
 
 const TabList = styled.ol`
     width: 19rem;
     list-style: none;
+
+    @media ${device.tabletL} {
+        width: 100%;
+    }
 `
 
 const TabContent = styled.div`
     flex: 1;
 `
 
 const StyledTab = styled.li`
-    cursor: pointer;
```

Comment:
why are we removing cursor?

Reply:
I can't remember why I removed it :D
I'll turn it back.